src/cljs/athens/views/blocks.cljs

```diff
@@ -432,7 +433,22 @@
       (swap! state assoc :string/local (.. e -target -value)))))
 
 
+(defn get-block-refs
+  [uid]
+  (d/q '[:find [?refs ...]
+         :in $ ?uid
+         :where
+         [?e :block/uid ?uid]
+         [?e :block/refs ?refs]]
+       @db/dsdb
+       uid))
```

Comment:
Move to `db.cljs`? This will be used when zoomed in on a block that has references.

Demo:

![image](https://user-images.githubusercontent.com/8952138/91784118-1407c580-ebd0-11ea-969e-9e2798283d0b.png)